third_party/istio-1.0.1/istio-lean.yaml

```diff
@@ -283,6 +283,162 @@ data:
       # Address where istio Pilot service is running
       discoveryAddress: istio-pilot.istio-system:15007
 
+---
+# Source: istio/templates/sidecar-injector-configmap.yaml
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: istio-sidecar-injector
```

Comment:
Should this be here?

Reply:
Yes, they have other injection mechanism that should still work even if automatic injection isn't turned on.

https://github.com/istio/istio/commit/17fd8e952edb05999a76185c9c50d71d6000daec

Reply:
Seems there is an option omitSidecarInjectorConfigMap added in the same commit, I'm going to use that.

Reply:
Done